lib/rules/yoda.js

```diff
@@ -73,14 +92,22 @@ function getNormalizedLiteral(node, defaultValue) {
         return node;
     }
 
-    if (looksLikeLiteral(node)) {
+    if (isNegativeNumericLiteral(node)) {
         return {
             type: "Literal",
             value: -node.argument.value,
             raw: `-${node.argument.value}`
         };
     }
 
+    if (isStaticTemplateLiteral(node)) {
+        return {
+            type: "Literal",
+            value: node.quasis[0].value.cooked,
+            raw: node.quasis[0].value.raw
```

Comment:
I guess it would be consistent to wrap this in backticks because string literal's `raw` has quotes (maybe even more correct to get token's `value`, but I think it isn't necessary).

Also, ``raw: `-${node.argument.value}` `` above should be probably ``raw: `-${node.argument.raw}` `` to retain the notation from the source code. Additionally,`` `-${node.argument.value}` `` is wrong for BigInt values.

Though, it seems that this property isn't used for anything in this rule? Not sure should we fix it and keep for possible future use in the message or rather remove it.

Reply:
This can be as well tweaked later, if and when the property becomes used for something.

Reply:
@mdjermanovic 
> This can be as well tweaked later, if and when the property becomes used for something.

Yes! I agree Thanks 👍